### PR TITLE
feat: Created Note App Template first in React

### PR DESCRIPTION
**Pull request by mdarif6**
feat: Created Note App Template first in React
- created note app pages in react.

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
clean code 
nice folder structure
suggestion :
instead of rendering header & footer on each page you can add them directly on App.js

everything else looks good to merge

File: src/pages/signup-page/SignupPage.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+html,
+:root {
+  --typora-primary-color: #7c3aed;
+}
+
+body {
+  background-color: lightgray;
```
Comment (PeeyushPrashant) [inline src/pages/signup-page/SignupPage.css]: provide variables as "var(--lightgrey)"

**Review by PeeyushPrashant — APPROVED** [review]
code structure looks good

just try attaching a screenshot while making pull request as it helps the reviewer to visualise

File: src/pages/archive-page/ArchiveAside.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                <select class="select" name="priority" id="priority">
+                  <option value="high">High</option>
+                  <option value="medium">Medium</option>
+                  <option value="low">Low</option>
+                </select>
+              </div>
+            </div>
+            <div class="selection">
+              <label className="lable-name">Select Label</label>
+              <div class="select-checkbox">
```
Comment (SJTGSHIVAM) [inline src/pages/archive-page/ArchiveAside.jsx]: these classes are everywhere let me know if you need help in fixing these. I can show you a way to fix it in one go.

File: src/pages/home-page/HomePage.css [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+  gap: 2rem;
+  margin: 2rem;
+}
+.aside-content-label {
+  display: flex;
+  gap: 2rem;
+}
+
+.aside-content-label,
+.aside-content-label i:hover {
```
Comment (SJTGSHIVAM) [inline src/pages/home-page/HomePage.css:56]: I really like the way you wrote this one

**Review by SJTGSHIVAM — COMMENTED** [review]
Hey Arif I believe merging it without fixing the class issue is not a good idea. let me know if you need help there. You can ping me for help anytime.

File: src/pages/signup-page/SignupPage.css (hunk 1 above)
Comment (mdarif6) [reply src/pages/signup-page/SignupPage.css]: same has been done

File: src/pages/archive-page/ArchiveAside.jsx (hunk 2 above)
Comment (mdarif6) [reply src/pages/archive-page/ArchiveAside.jsx]: thanks, class=className correction has done

**Pull request merged**